draft-ietf-tls-tls13.md

```diff
@@ -3059,7 +3059,7 @@ Alert messages ({{alert-protocol}}) MUST NOT be fragmented across records.
            ContentType type;
            ProtocolVersion legacy_record_version = { 3, 1 };    /* TLS v1.x */
            uint16 length;
-           opaque fragment[TLSPlaintext.length];
+           opaque fragment[length];
```

Comment:
While technically true, the "length" here is defined one line above. Adding in the object explicitly within the object itself just makes it a little more complicated than needed. I'd leave this one as-is.